test/functional/tests/io_class/test_io_class_file.py

```diff
@@ -385,26 +391,28 @@ def test_ioclass_file_size(filesystem):
         test_files = []
         for size, ioclass_id in size_to_class.items():
             occupancy_before = cache.get_io_class_statistics(
-                io_class_id=ioclass_id).usage_stats.occupancy
+                io_class_id=ioclass_id
+            ).usage_stats.occupancy
             file_path = f"{mountpoint}/test_file_{size.get_value()}"
             Dd().input("/dev/zero").output(file_path).oflag("sync").block_size(size).count(1).run()
             sync()
             drop_caches(DropCachesMode.ALL)
             occupancy_after = cache.get_io_class_statistics(
-                io_class_id=ioclass_id).usage_stats.occupancy
+                io_class_id=ioclass_id
+            ).usage_stats.occupancy
             if occupancy_after != occupancy_before + size:
-                TestRun.fail("File not cached properly!\n"
-                             f"Expected {occupancy_before + size}\n"
-                             f"Actual {occupancy_after}")
+                TestRun.fail(
+                    "File not cached properly!\n"
+                    f"Expected {occupancy_before + size}\n"
+                    f"Actual {occupancy_after}"
+                )
             test_files.append(File(file_path).refresh_item())
         sync()
         drop_caches(DropCachesMode.ALL)
 
     with TestRun.step("Move all files to 'unclassified' IO class."):
-        ioclass_config.remove_ioclass_config(ioclass_config_path=ioclass_config_path)
-        ioclass_config.create_ioclass_config(
-            add_default_rule=False, ioclass_config_path=ioclass_config_path
-        )
+        ioclass_config.remove_ioclass_config(ioclass_config_path)
+        ioclass_config.create_ioclass_config(False, ioclass_config_path)
```

Comment:
Actually, it was clearer with keyword arguments

Reply:
This change is part of reformatting and aligning code across files fore some consistence (this function is called without keyword arguments in other places already).
I think it would be best to leave discussion about this function api/readablity and coding standards for separate patch as it will affect files from other test group.